cc430-rtc.c: wrong function attributes

> cc430-rtc.c:177:46: warning: function requires 8 bytes for stack storage but frame allocation inhibited by 'naked' [-Wattributes]

Should that rather be `no_instrument_function` instead of `naked`?

--- a/cpu/cc430/cc430-rtc.c
+++ b/cpu/cc430/cc430-rtc.c
@@ -197,7 +197,7 @@ interrupt(RTC_VECTOR) __attribute__((naked)) rtc_isr(void)
         }
 
         if (rtc_second_pid) {
-            msg_t m;
+            static msg_t m;
             m.type = RTC_SECOND;
             msg_send_int(&m, rtc_second_pid);
         }

